src/js/incognito.js

```diff
@@ -8,28 +8,23 @@ chrome.tabs.query({}, function(results) {
   });
 });
 
-// Create tab event listeners
-function onUpdatedListener(tabId, changeInfo, tab) {
+// Tab event handlers
+function onUpdatedHandler(tab) {
   tabs[tab.id] = tab.incognito;
 }
 
-function onRemovedListener(tabId) {
+function onRemovedHandler(tabId) {
   delete tabs[tabId];
 }
 
-// Subscribe to tab events
-function startListeners() {
-  chrome.tabs.onUpdated.addListener(onUpdatedListener);
-  chrome.tabs.onRemoved.addListener(onRemovedListener);
-}
-
 function tabIsIncognito(tabId) {
   return tabs[tabId] || false;
 }
 
 /************************************** exports */
 var exports = {};
-exports.startListeners = startListeners;
+exports.onRemoved = onRemovedHandler;
```

Comment:
Can we keep than idiom `exports.sameName = sameName`? It makes things easier to find with `grep`.